### ISSUE TITLE
Integrace kalendáře z mrak.pirati.cz

### ISSUE THREAD

**Issue by jcz1** [pirati-web/jekyll-theme-pirati#38]
Prosíme o možnost integrace kalendáře z NextCloudu (mrak.pirati.cz). Ten mimo jiné umožňuje embedding pomocí iframe.

**Comment by b0rmann** [comment]
CSP policy na mraku jsou nastavene. iframe se da embedovat z *.pirati.cz

**Comment by jcz1** [comment]
Super. Mohl by někdo připravit kód do témátka? Pak se můžeme zamyslet jak přenést data z googlu.

**Comment by xaralis** [comment]
@b0rmann Má to nějaký api? Jestli ne tak do té doby třeba pro nás nepoužitelný, protože to vypadá fakt odporně 😔

**Comment by jcz1** [comment]
P.S. Nějaký kód má asi na webu Tomáš Vymazal, ale nevím, kde mu to běží.

**Comment by b0rmann** [comment]
@jcz1 to je jen iframe.

**Comment by b0rmann** [comment]
@xaralis no, v pripade sdileni kalendare s toho se da vytahnout r/o **ical**
napr: https://mrak.pirati.cz/remote.php/dav/public-calendars/ATFSt9ZkZ3WHbyzo?export

jinak, samozrejme, to ma caldav rozhrani a da se na nej napojit jakykoliv klient (ale to uz je s autorizaci) https://mrak.pirati.cz/remote.php/dav/

**Comment by xaralis** [comment]
To je k prdu. Normální REST nemá, jo? Aktuálně na gcal ti stačí token s patřičnými právy a můžeš využívat normální [Google Calendar API](https://developers.google.com/calendar/).

**Comment by b0rmann** [comment]
pro kalendare neni normou (standartem) REST, ale ical
